lib/Drawers/Drawer.js

```diff
@@ -182,7 +182,8 @@ var MoonDrawer = module.exports = kind(
 	*/
 	handlers: {
 		ontransitionend: 'transitionEnded',
-		onwebkittransitionend: 'transitionEnded'
+		onwebkittransitionend: 'transitionEnded',
+		onDrawerAnimationStep: 'updateControlDrawersHeight'
```

Comment:
Every single animation step triggers  `onDrawerAnimationStp` event. 
I do not think it is a good way for performance.